_includes/pbc/all/install-features/202304.0/install-glue-api/install-the-picking-app-glue-api.md

```diff
@@ -0,0 +1,197 @@
+| UserBackendApi          | vendor/spryker/user-backend-api           |
+| WarehouseUserBackendApi | vendor/spryker/warehouse-user-backend-api |
+
+{% endinfo_block %}
+
+## 2) Set up configuration
+
+Adjust the protected paths configuration if you want to make `warehouse-user-assignments` resource protected:
+
+**src/Pyz/Shared/GlueBackendApiApplicationAuthorizationConnector/GlueBackendApiApplicationAuthorizationConnectorConfig.php**
+
+```php
+<?php
+
+namespace Pyz\Shared\GlueBackendApiApplicationAuthorizationConnector;
+
+use Spryker\Shared\GlueBackendApiApplicationAuthorizationConnector\GlueBackendApiApplicationAuthorizationConnectorConfig as SprykerGlueBackendApiApplicationAuthorizationConnectorConfig;
+
+class GlueBackendApiApplicationAuthorizationConnectorConfig extends SprykerGlueBackendApiApplicationAuthorizationConnectorConfig
+{
+    /**
+     * @return array<string, mixed>
+     */
+    public function getProtectedPaths(): array
+    {
+        return [
+            '/\/warehouse-user-assignments(?:\/[^\/]+)?\/?$/' => [
+                'isRegularExpression' => true,
+            ],
+        ];
+    }
+}
+
+```
+
+## 3) Set up transfer objects
+
+Generate transfers:
+```bash
+vendor/bin/console transfer:generate
+```
+
+{% info_block warningBox "Verification" %}
+
+Ensure that the following changes have occurred in transfer objects:
+
+| TRANSFER                               | TYPE  | EVENT   | PATH                                                                 |
+|----------------------------------------|-------|---------|----------------------------------------------------------------------|
+| WarehouseUserAssignmentsRestAttributes | class | created | src/Generated/Shared/Transfer/WarehouseUserAssignmentsRestAttributes |
+| WarehousesRestAttributes               | class | created | src/Generated/Shared/Transfer/WarehousesRestAttributes               |
+
+{% endinfo_block %}
+
+## 4) Set up behavior
+
+Set up the following behaviors.
+
+Activate the following plugin:
+
+| PLUGIN                                                  | SPECIFICATION                                                                             | PREREQUISITES | NAMESPACE                                                   |
+|---------------------------------------------------------|-------------------------------------------------------------------------------------------|---------------|-------------------------------------------------------------|
+| WarehouseUserAssignmentsResourcePlugin                  | Registers the `warehouse-user-assignments` resource.                                      |               | Spryker\Glue\WarehouseUserBackendApi\Plugin\GlueApplication |
+| UserByWarehouseUserAssignmentResourceRelationshipPlugin | Adds the `users` resource as a relationship to the `warehouse-user-assignments` resource. |               | Spryker\Glue\UserBackendApi\Plugin\GlueJsonApiConvention    |
```

Comment:
This plugin belongs to the connection of two features:
- Backoffice
- Warehouse User Management